tests/acceptance/test_async.py

```diff
@@ -39,3 +44,83 @@ async def product_task(a, b):
 
     assert sum_results == [3, 7, 11]
     assert product_results == [12]
+
+
+async def test_cancel(async_app):
+    sum_results = []
+
+    @async_app.task(queue="default", name="sum_task")
+    async def sum_task(a, b):
+        sum_results.append(a + b)
+
+    job_id = await sum_task.defer_async(a=1, b=2)
+    await sum_task.defer_async(a=3, b=4)
+
+    await async_app.job_manager.cancel_job_by_id_async(job_id)
+
+    status = await async_app.job_manager.get_job_status_async(job_id)
+    assert status == Status.CANCELLED
+
+    jobs = await async_app.job_manager.list_jobs_async()
+    assert len(jobs) == 2
+
+    await async_app.run_worker_async(queues=["default"], wait=False)
+
+    assert sum_results == [7]
+
+
+async def test_cancel_with_delete(async_app):
+    sum_results = []
+
+    @async_app.task(queue="default", name="sum_task")
+    async def sum_task(a, b):
+        sum_results.append(a + b)
+
+    job_id = await sum_task.defer_async(a=1, b=2)
+    await sum_task.defer_async(a=3, b=4)
+
+    await async_app.job_manager.cancel_job_by_id_async(job_id, delete_job=True)
+
+    jobs = await async_app.job_manager.list_jobs_async()
+    assert len(jobs) == 1
+
+    await async_app.run_worker_async(queues=["default"], wait=False)
+
+    assert sum_results == [7]
+
+
+async def test_abort(async_app):
+    @async_app.task(queue="default", name="task1", pass_context=True)
+    async def task1(context):
+        while True:
+            await asyncio.sleep(1)
```

Comment:
Computers are fast. Let's wait much less than this unless we need to wait for this much.

https://www.youtube.com/watch?v=6xijhqU8r2A

Suggestion: 0.1s